components/opponent/wikis/warcraft/player_ext_custom.lua

```diff
@@ -110,10 +110,18 @@ function CustomPlayerExt.syncPlayer(player, options)
 	return player
 end
 
+---Same as PlayerExt.syncPlayer, except it does not save the player's flag to page variables.
+---@param player WarcraftStandardPlayer
+---@param options {fetchPlayer: boolean, fetchMatch2Player: boolean, date: string?}?
+---@return WarcraftStandardPlayer
+function CustomPlayerExt.populatePlayer(player, options)
+	return CustomPlayerExt.syncPlayer(player, Table.merge(options, {savePageVar = false}))
+end
```

Comment:
Why is this needed or wanted?

Reply:
so that when `populatePlayer` is called the `syncPlayer` from the custom is called
technically it currently is a bug